### PR TITLE
use /tmp for osx cluster up

### PR DESCRIPTION
**Pull request by mfojtik**
@deads2k docker for mac only mounts /Users, /Volumes, /private and /tmp by default... 

However:

```
$ echo $TMPDIR
/var/folders/jm/0vnz7tm15tl9g9n90cwqb3c00000gn/T/
```

That means golang ioutil.TempDir() use os.TempDir which will write all configs to location that cannot be mounted into container.... Using `/tmp` as os.TempDir in OSX seems to fix this.

### CODE REVIEW

File: pkg/oc/bootstrap/docker/openshift/helper.go [hunk 1]
```diff
@@ -577,7 +578,11 @@ func masterHTTPClient(localConfig string) (*http.Client, error) {
 // copyConfig copies the OpenShift configuration directory from the
 // server directory into a local temporary directory.
 func (h *Helper) copyConfig() (string, error) {
-	tempDir, err := ioutil.TempDir("", "openshift-config")
+	tmpDirFn := os.TempDir
+	if runtime.GOOS == "darwin" {
+		tmpDirFn = func() string { return "/tmp" }
```
Comment (sttts) [inline pkg/oc/bootstrap/docker/openshift/helper.go]: please add a comment

File: pkg/oc/bootstrap/docker/openshift/helper.go [hunk 2]
```diff
@@ -577,7 +578,12 @@ func masterHTTPClient(localConfig string) (*http.Client, error) {
 // copyConfig copies the OpenShift configuration directory from the
 // server directory into a local temporary directory.
 func (h *Helper) copyConfig() (string, error) {
```
Comment (deads2k) [inline pkg/oc/bootstrap/docker/openshift/helper.go]: huh, I didn't think I was touching this anymore.

**Comment by deads2k** [comment]
@mfojtik sorry, I rebased under this.  Rebase it on my branch and I'll merge

**Comment by deads2k** [comment]
picked.

**Pull request closed without merging**